src/tapedfunction.jl

```diff
@@ -1,10 +1,6 @@
 abstract type AbstractInstruction end
-
-mutable struct Tape
-    tape::Vector{<:AbstractInstruction}
-    counter::Int
-    owner
-end
+abstract type Taped end
+const Tape = Vector{AbstractInstruction}
```

Comment:
Currently it is `Vector{<:AbstractInstruction}`, is this change intended or shoild it be
```suggestion
const Tape = Vector{<:AbstractInstruction}
```

Reply:
But I can't create an instance of `Vector{<:AbstractInstruction}` by `Vector{<:AbstractInstruction}()`, is there a way to deal with this?

Reply:
Maybe use a different dedicated function for whatever you want to do?

Reply:
@KDr2 I suggest that we remove the immediate type `Tape` and work with `Vector{AbstractInstruction}` directly.